src/lib/content/types.ts

```diff
@@ -177,7 +177,11 @@ export interface UpdateContentPayload {
       longitude?: string
     },
     logo_url?: string,
-    business_hours?: BusinessHours,
+    busines_hours?: Array<{
```

Comment:
Should we just do `Array<BusinessHours>`? or we can leave it as is,  but maybe delete the BusinessHours type?